### PR TITLE
[DNM] ED-3115 add env vars to makefile

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/ED-3115

### CODE REVIEW

File: app/navigator/settings/base.py [hunk 1]
```diff
@@ -228,36 +221,9 @@
 SSO_PROXY_SESSION_COOKIE = os.environ.get(
     'SSO_PROXY_SESSION_COOKIE', 'debug_sso_session_cookie')
 
+HEADER_FOOTER_CONTACT_US_URL = os.environ.get('HEADER_FOOTER_CONTACT_US_URL')
```
Comment (richtier) [inline app/navigator/settings/base.py]: we want to default to the production value.

That way, if we forget the set the env var in prod then things will still work.

File: app/navigator/settings/base.py [hunk 2]
```diff
@@ -228,36 +221,9 @@
...
-SERVICES_EXOPPS = os.getenv('SERVICES_EXOPPS')
-SERVICES_FAB = os.getenv('SERVICES_FAB')
-SERVICES_GET_FINANCE = os.getenv('SERVICES_GET_FINANCE')
-SERVICES_SOO = os.getenv('SERVICES_SOO')
-
-# FOOTER LINKS
-INFO_ABOUT = os.getenv('INFO_ABOUT')
-INFO_CONTACT_US_DIRECTORY = os.getenv('INFO_CONTACT_US_DIRECTORY')
-INFO_PRIVACY_AND_COOKIES = os.getenv('INFO_PRIVACY_AND_COOKIES')
-INFO_TERMS_AND_CONDITIONS = os.getenv('INFO_TERMS_AND_CONDITIONS')
```
Comment (richtier) [inline app/navigator/settings/base.py]: we need all these, no?

File: app/navigator/settings/base.py [hunk 3]
```diff
@@ -83,13 +83,6 @@
     },
 ]
 
-HEADER_FOOTER_CONTACT_US_URL = os.getenv(
```
Comment (richtier) [inline app/navigator/settings/base.py]: we want this to be defaulted to prod value, so the prod app will work even if we forget to set the env var.

Comment (eadpearce) [reply app/navigator/settings/base.py]: this is actually an outdated var in directory-header-footer - it's not used in templates. we should remove it from there

**Pull request closed without merging**